config/routes.rb

```diff
@@ -53,6 +53,7 @@
         get 'customers' => :customers
       end
     end
+    resources :currencies
```

Comment:
whitelist only those that are being used

Reply:
done